_includes/components/homepage/Revenue/partials/Item.jsx

```diff
@@ -2,13 +2,13 @@ import { h } from 'preact';
 import trimValues from './trimValues.js';
 
 
-export default function Item({ title, value }) {
+export default function Item({ year, title, value }) {
   return (
     <div className="Revenue-itemWrap">
-      <div className="Revenue-item">
+      <a href={`/${year}/search-result?search_type=full-search&search=${encodeURI(title)}`} className="Revenue-item">
```

Comment:
remember we want non-alphanum to be replaced with space. I suggested that because ckan doesn't tolerate commas and things well right now, and this is a front-and-centre feature. I'll add something to pivotal to make search more tolerant.

Reply:
looks like commas are alright - if you tested with all the functions, uriencode is probably sufficient.

Reply:
Ok. Will do.